xbmc/cores/VideoPlayer/VideoRenderers/RenderManager.cpp

```diff
@@ -891,13 +894,14 @@ void CRenderManager::UpdateLatencyTweak()
 {
   float fps = CServiceBroker::GetWinSystem()->GetGfxContext().GetFPS();
   const bool isHDREnabled = CServiceBroker::GetWinSystem()->GetOSHDRStatus() == HDR_STATUS::HDR_ON;
+  const bool isHDRUsed = isHDREnabled && (m_picture.hdrType != StreamHdrType::HDR_TYPE_NONE);
```

Comment:
Suggestion, something like:
```
 const HDR_STATUS status = CServiceBroker::GetWinSystem()->GetOSHDRStatus();
 const bool isHDRUsed = (status == HDR_STATUS::HDR_UNSUPPORTED)
                            ? (m_picture.hdrType != StreamHdrType::HDR_TYPE_NONE)
                            : (status == HDR_STATUS::HDR_ON);
```
or the lengthier version without ? ;

Reply:
This is not correct anymore. Now it would be `isHDRUsed == true` when HDR is not supported by hardware!?

Reply:
No, it's not about the syntax. You don't get to redefine the meaning of HDR_ON.
The API needs to be modified for what you want to do, see 3/ and 4/ in my last comment.
The API change would be useful for other platforms too. If implemented and needed by at least one Kodi platform, the merge won't be a problem and CE can piggy-back.
I don't know which platform is the lowest hanging fruit that maybe you could contribute, if no team dev shows interest.

Reply:
On Windows different latency ONLY depends if HDR mode is enabled (HDR_STATUS::HDR_ON)

e.g. 
4K SDR video converted to HDR (by Windows) has same latency that 4K native HDR video.

Then effectively the only condition `CServiceBroker::GetWinSystem()->GetOSHDRStatus() == HDR_STATUS::HDR_ON` is enough to apply special latency.

Is not relevant if source is HDR or not (`m_picture.hdrType`) 

As GetOSHDRStatus() can be implemented as "override" for every platform in windowing,  the implementation can be changed for Linux only (to certain extent) without having to change things in RenderManager (which affects all platforms).

Reply:
Another posibility is create new "API" like

`VideoSourceIsPlayableInHDR()`

and in Windows override with always "true"...

While in Linux as:
`return m_picture.hdrType != StreamHdrType::HDR_TYPE_NONE;`

Then in RenderManager:
```
isHDRUsed = (GetOSHDRStatus() == HDR_STATUS::HDR_ON) && VideoSourceIsPlayableInHDR();
```

Reply:
> 4K SDR video converted to HDR (by Windows)

In my opinion this is the place the start. By such case the user is forced to watch SDR in HDR. We have seen in past it's only a small amount of users want to watch SDR in HDR as the result is very dependent by users hardware. This is why we use a user setting with default off converting the original media. And I think such option to auto convert SDR to HDR is very platform dependent and maybe not available for everyone.

Reply:
Likewise, an HDR video can be played in SDR on Windows even if the TV supports HDR. 

On Windows, all combinations are possible, but again, the only condition that matters for latency is `HDR_STATUS::HDR_ON`.

Reply:
> because user has forced it from Windows settings

Then this option must be incount as user can disable it and watch SDR on HDR screen.

Reply:
But Windows users are complicated :)

https://www.reddit.com/r/kodi/comments/rhmczb/comment/jqoyq2b/?context=3

https://forum.kodi.tv/showthread.php?tid=366975

https://github.com/xbmc/xbmc/pull/23509

Reply:
Users are same on all platforms 😇.
This is the reason default is SDR is SDR. User can switch himself if he prefer to convert it. Each media should be used in his desired purpose. It does also not make my Opel better when I place a Ferrari sticker on it 😁.